mitodl/bootcamp-ecommerce
# Add admins for models

#### What are the relevant tickets?
Fixes #24

#### What's this PR do?
Adds Django admin interfaces to models

#### How should this be manually tested?
Go to `/admin` and verify they exist. The order audit table should not be modifiable
